### PR TITLE
Spherical branch

### PR DESCRIPTION
**Pull request by SphericalCat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3978 +1,7 @@
-# Kottans-frontend
```
Comment (suchov) [inline README.md:1]: You deleted 3979 lines in README.md - kind of decided to leave only your comment there? :rofl:

**Review by suchov — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SphericalCat) [reply README.md:1]: i`m so sorry😭

File: README.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+<<<<<<< HEAD
```
Comment (suchov) [inline README.md:1]: Wonderful.
Let's just get rid of some lines that should be deleted when resolving conflicts like this one.
The lines that contains <<<<<< >>>>>>> ====== and the name of branches should be removed.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+**Hi, i`m Maryna** <img src="paw.png" width="35">
+
+Meow meow!! Meow, meow meow meow meow. Meow meow meow, meow meow!
+![](6kn.gif)
+
+# Kottans-frontend
+=======
```
Comment (suchov) [inline README.md:8]: Should be removed too.

File: README.md [hunk 4]
```diff
@@ -3999,3 +4007,4 @@ Hello, kottans!
 
 ## Hello Everyone!
 Happy to be here:)
+>>>>>>> master
```
Comment (suchov) [inline README.md:4010]: should be removed.

**Review by suchov — CHANGES_REQUESTED** [review]
And the "mosk-repo" folder do we really need it?

File: README.md (hunk 4 above)
Comment (SphericalCat) [reply README.md]: Corrected, please take a look

**Comment by suchov** [comment]
The number of changes is right now - 3 files, but the are a lot of green and red lines - that shows that you changed a lot of lines.
Is it on purpose, or just your IDE playing with you? Check the discord for task 0 pls.

**Pull request closed without merging**